File: a111(feynman)/C++/feynman.cpp
#include <iostream>


int GetAmountOfSquare(const int n) {
  return (n * (n + 1) * (2 * n + 1)) / 6;
}


int main(int argc, char const *argv[]) {
  int n = 0;
  while (std::cin >> n) {
    if (!n) {
      break;
    }
    std::cout << GetAmountOfSquare(n) << '\n';
  }
  return 0;
}

File: a111(feynman)/Python/feynman.py
from sys import stdin


def get_amount_of_square(n: int) -> int:
    return (n * (n + 1) * (2 * n + 1)) // 6


for n in map(int, stdin):
    if not n:
        break
    print(get_amount_of_square(n))
